ansible/module_utils/port_utils.py

```diff
@@ -198,6 +198,11 @@ def get_port_alias_to_name_map(hwsku, asic_id=None):
         else:
             for i in range(1,9):
                 port_alias_to_name_map["Ethernet1/%d" % i] = "Ethernet%d" % ((i - 1) * 4)
+    elif hwsku == "B6510-48VS8CQ":
+        for i in range(1,48):
+            port_alias_to_name_map["twentyfiveGigE0/%d" % i] = "Ethernet%d" % i
+        for i in range(49,56):
```

Comment:
range(49,56) to include 56 as well

Reply:
The device port from Ethernet49 to Ethernet 56 are 100G.

Reply:
The same comment as above. should be range(49,57)

Reply:
Fixed , retesting.